exalearn-ui/src/app/components/question/question.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Question } from '../../interfaces/interfaces';
+
+@Component({
+	selector: 'app-question',
+	templateUrl: './question.component.html',
+	styleUrls: ['./question.component.scss']
+})
+export class QuestionComponent implements OnInit {
+	// @ts-ignore
```

Comment:
Do we really need ts-ignore?

Reply:
I cannot use @Input normally because tsconfig (or lint) requires assgning to every variable including inputs